ocs_ci/ocs/resources/pod.py

```diff
@@ -4013,3 +4014,42 @@ def wait_for_ceph_cmd_execute_successfully(
         f"The ceph command failed to execute successfully after {num_of_retries} retries"
     )
     return False
+
+
+def get_osd_pod_pvc(osd_pod):
```

Comment:
you can use this existing function instead https://github.com/red-hat-storage/ocs-ci/blob/3d8e5d8f68210023671c50e10f0ba3b81bca3035/ocs_ci/ocs/resources/pod.py#L1943

Reply:
I saw the function `get_pvc_name`, but this will not work with getting the osd PVC. Should I update the function with my changes of searching the first occurrence of the `persistentVolumeClaim`?

Reply:
I saw some cases in the osd PVC, which will not be the first occurrence.